tests/src/Common/test_dependencies/project.json

```diff
@@ -1,9 +1,9 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "2.0.0-preview1-25214-03",
-    "Microsoft.Private.CoreFx.NETCoreApp": "4.4.0-preview1-25214-03",
-    "System.Memory": "4.4.0-preview1-25214-03",
-    "System.Runtime.CompilerServices.Unsafe": "4.4.0-preview1-25214-03"
+    "Microsoft.NETCore.Platforms": "2.0.0-preview1-25220-02",
+    "Microsoft.Private.CoreFx.NETCoreApp": "4.4.0-preview1-25220-02",
+    "System.Memory": "4.4.0-preview1-25220-02",
+    "System.Runtime.CompilerServices.Unsafe": "4.4.0-preview1-25220-02"
```

Comment:
This auto-update PR is against release/2.0.0, which forked off before that package was added to master with https://github.com/dotnet/coreclr/commit/737162181ad18b5b83e951ee910ff513e3283d1d.